PlaceRouteHierFlow/placer/SeqPair.cpp

```diff
@@ -1173,18 +1173,18 @@ std::string SeqPair::getLexIndex(design& des) const {
 bool SeqPair::CheckSymm(design& caseNL) {
   auto logger = spdlog::default_logger()->clone("placer.SeqPair.CheckSymm");
   std::map<int, int> posPosition, negPosition;
-  for (int i = 0; i < posPair.size(); ++i) {
```

Comment:
The usual fix for this is to make the int an unsigned instead of casting the size into an int.